<launch>
  <arg name="mav_name" default="euroc6"/>
  <arg name="namespace" default="$(arg mav_name)"/>
  <arg name="broadcast_ip" default="10.10.1.255"/>

  <!-- Vicon VRPN-client -->
  <include file="$(find ros_vrpn_client)/launch/asl_vicon.launch">
    <arg name="object_name" value="$(arg mav_name)" />
  </include>

<group ns="$(arg namespace)">
  <node name="vicon_transmitter" pkg="euroc_vicon_transmission" type="vicon_transmitter_node" output="screen">
    <rosparam command="load" file="$(find euroc_vicon_transmission)/cfg/vicon_transmitter.yaml" />
    <param name="broadcast_ip" type="string" value="$(arg broadcast_ip)" />
  </node>
</group>

</launch>
